server/src/main/java/org/candlepin/liquibase/PerOrgProductsMigrationTask.java

```diff
@@ -0,0 +1,455 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.liquibase;
+
+import liquibase.database.Database;
+import liquibase.exception.DatabaseException;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+
+/**
+ * The PerOrgProductsMigrationTask performs the post-db upgrade data migration to the cp2_* tables.
+ */
```

Comment:
Could probably use a description of what this is actually going to do. For example, what is it going to do and why?